manual/manual/html_processing/Makefile

```diff
@@ -1,13 +1,84 @@
 DUNE_CMD := $(if $(wildcard dune/dune.exe),dune/dune.exe,dune)
 DUNE ?= $(DUNE_CMD)
 
-all:
-	$(DUNE) exec --root=. src/process_manual.exe quiet
-	$(DUNE) exec --root=. src/process_api.exe overwrite quiet
-	$(DUNE) exec --root=. src/process_api.exe compiler overwrite quiet
+DEBUG ?= 0
+ifeq ($(DEBUG), 1)
+    DBG=
+else
+    DBG=quiet
+endif
+
+WEBDIR = ../webman
+WEBDIRMAN = $(WEBDIR)/manual
+WEBDIRAPI = $(WEBDIR)/api
+WEBDIRCOMP = $(WEBDIRAPI)/compilerlibref
+
+# The "all" target generates the Web Manual in the directories
+# ../webman/manual, ../webman/api, and ../webman/api/compilerlibref
+all: css js img
+	$(DUNE) exec --root=. src/process_manual.exe $(DBG)
+	$(DUNE) exec --root=. src/process_api.exe overwrite $(DBG)
+	$(DUNE) exec --root=. src/process_api.exe compiler overwrite $(DBG)
+
+$(WEBDIR):
+	mkdir -p $(WEBDIRMAN)
+	mkdir -p $(WEBDIRCOMP)
+
+$(WEBDIRMAN)/manual.css: scss/_common.scss scss/manual.scss $(WEBDIR)
+	sass scss/manual.scss > $(WEBDIRMAN)/manual.css
+
+$(WEBDIRAPI)/style.css: scss/_common.scss scss/style.scss $(WEBDIR)
+	sass scss/style.scss > $(WEBDIRAPI)/style.css
+	cp $(WEBDIRAPI)/style.css $(WEBDIRCOMP)/style.css
+
+css: $(WEBDIRMAN)/manual.css $(WEBDIRAPI)/style.css
+
+# Just copy the JS files
+JS_FILES0 := scroll.js search.js
+JS_FILES := $(addprefix $(WEBDIRAPI)/, $(JS_FILES0)) $(addprefix $(WEBDIRCOMP)/, $(JS_FILES0)) $(WEBDIRMAN)/scroll.js
+# should be same as:
+#JS_FILES := $(WEBDIRAPI)/scroll.js $(WEBDIRAPI)/search.js $(WEBDIRCOMP)/scroll.js $(WEBDIRCOMP)/search.js $(WEBDIRMAN)/scroll.js
+
+# There must be a more clever way
+$(WEBDIRAPI)/%.js: js/%.js
+	cp $< $@
+
+$(WEBDIRMAN)/%.js: js/%.js
+	cp $< $@
+
+$(WEBDIRCOMP)/%.js: js/%.js
+	cp $< $@
+
+js: $(WEBDIR) $(JS_FILES)
+
+# Copy images
+IMG_FILES0 := colour-logo-gray.svg favicon.ico
+IMG_FILES1 := $(IMG_FILES0) search_icon.svg
+IMG_FILES := $(addprefix $(WEBDIRAPI)/, $(IMG_FILES1)) $(addprefix $(WEBDIRCOMP)/, $(IMG_FILES1)) $(addprefix $(WEBDIRMAN)/, $(IMG_FILES0)) 
+
+# There must be a more clever way
```

Comment:
Could possibly use the trick above...

Reply:
in view of your previous comment, let's leave it as it for now